src/components/common/Dialogs/CreateDomainDialog/CreateDomainDialog.tsx

```diff
@@ -86,7 +88,7 @@ const CreateDomainDialog = ({
           }
           return (
             <CreateDomainDialogForm
-              back={prevStep ? () => callStep(prevStep) : undefined}
+              back={prevStep && callStep ? () => callStep(prevStep) : undefined}
```

Comment:
I think this might be a type issue here, as you shouldn't need to check `callStep` for existence

Reply:
This is a result of me making `WizardDialogType<object>` Partial because now this Dialog can be called from inside the colony domain selector, which doesn't provide `prevStep` nor a `callStep` prop because it's not the wizard we use for the UAC.